dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsResource.java

```diff
@@ -6,8 +6,8 @@
 import com.dotcms.rest.ResponseEntityView;
 import com.dotcms.rest.WebResource;
 import com.dotcms.rest.annotation.NoCache;
-import com.dotcms.rest.api.v1.authentication.ResponseUtil;
 import com.dotcms.rest.api.v1.apps.view.AppView;
+import com.dotcms.rest.api.v1.authentication.ResponseUtil;
```

Comment:
Changing this file on this PR is not necessary as the remove orphan endpoint was deleted